docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  web:
+    image: interactivethings/electricity-prices-switzerland
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.web.rule=PathPrefix(`/`)"
+      - "traefik.http.services.web.loadbalancer.server.port=3000"
+
+  screenshot:
+    image: interactivethings/electricity-prices-switzerland-screenshot
+    # Due to the image being based on node, we need to use init
+    # to ensure the container correctly handles signals
+    # See https://github.com/nodejs/docker-node/blob/7f99010af378daa2520bb60435d0e91e20950d82/docs/BestPractices.md#handling-kernel-signals
```

Comment:
Great catch @ptbrowne 🎉 Would be cool to share this insight with the team to help make clean handling of signals more standardized across projects.